app/controllers/casework/maat_decisions_controller.rb

```diff
@@ -66,11 +66,7 @@ def destroy
 
       set_flash :decision_removed, success: true
 
-      if @crime_application.decision_ids.size > 1
```

Comment:
This is handled in the SendDecisionsController which redirects if there are no decisions.